google-cloud-spanner/src/test/java/com/google/cloud/spanner/it/ITClosedSessionTest.java

```diff
@@ -169,10 +170,11 @@ public void testReadOnlyTransaction() {
   public void testReadOnlyTransactionNoRecreation() {
     client.setAllowSessionReplacing(false);
     client.invalidateNextSession();
-    expectedException.expect(SessionNotFoundException.class);
     try (ReadOnlyTransaction txn = client.readOnlyTransaction()) {
       try (ResultSet rs = txn.executeQuery(Statement.of("SELECT 1"))) {
         rs.next();
```

Comment:
Add a `fail("Expected exception")` statement to prevent the test case succeeding in case of no error.